wger/exercises/templates/muscles/overview.html

```diff
@@ -45,52 +76,54 @@
 <div class="row">
     <div class="col-sm-4 col-xs-6">
         <h4 style="padding-bottom:0.5em;">{% trans "Muscles" %}</h4>
-        <div id="muscle-system"
-             class="muscle-background"
-             style="background-image: url('/static/images/muscles/muscular_system_front.svg');">
```

Comment:
Did you try the format `{% static 'images/muscles/muscular_system_front.svg' %}` here instead of passing in the whole `url`? Not sure if it'll work, try it and see. Might look better, plus it's more in line with Django templating style.

Reply:
No, I will try that @wangonya, thank you for the feedback.

Reply:
tried it but it is not working